### DISCUSSION TITLE
Blocks

### DISCUSSION THREAD

**Discussion by spurtcmsdeveloper-73** [spurtcms/spurtcms#88] [General]
We are excited in introducing a powerful feature in spurtcms admin the concept of blocks. These are modular units which allow flexibility in website creation blocks can be arranged and positioned within different regions of page layout. This allows flexibility and dynamic customization of pages.
![360383720-662865b0-c53f-46f6-87a9-30c1222ebd29](https://github.com/user-attachments/assets/5c31073c-2667-4390-8095-eecab32864e8)

**Comment by nithya-picco** [comment]
Where should I create blocks in the SpurtCMS admin panel to make use of the new feature effectively?

**Comment by alsherif-khalaf** [comment]
Can we develop new customized Blocks ?

**Reply by spurtcms-golang** [reply]
Yes, you can develope own Blocks , it has supports Tailwind css

**Reply by alsherif-khalaf** [reply]
> Yes, you can develope own Blocks , it has supports Tailwind css

I searched and I couldn't find any resource to learn how 
https://www.spurtcms.com/blocks-for-cms-website

**Reply by spurtcms-golang** [reply]
You can login in to admin panel , you can navigate to Blocks section using left menu.  

Click the button right cornor called **"Create Blocks"**

see the attached screen shot


<img width="1262" alt="Screen Shot 2025-02-11 at 8 04 54 AM" src="https://github.com/user-attachments/assets/c849241c-863c-4a5d-9f82-52404409b6e2" />


<img width="678" alt="Screen Shot 2025-02-11 at 8 05 06 AM" src="https://github.com/user-attachments/assets/88d422fd-4ec0-4c57-817f-fc9ea2bb59a6" />
